Cargo.toml

```diff
@@ -51,7 +51,7 @@ external-secp = []
 # All these dependencies must match the versions in:
 # https://github.com/zcash/zcash/blob/<git subtree version>/Cargo.toml
 blake2b_simd = "1"
-cxx = { version = "=1.0.79", features = ["c++17"] }
+cxx = { version = "=1.0.89", features = ["c++17"] }
```

Comment:
Needs to match `zcashd`, which is currently `1.0.83`:
https://github.com/zcash/zcash/blob/master/Cargo.toml#L71